web/src/engine/websites/Akuma.ts

```diff
@@ -0,0 +1,47 @@
+import { Tags } from '../Tags';
+import icon from './Akuma.webp';
+import { Chapter, DecoratableMangaScraper, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+const pageScript = `
+    new Promise( (resolve, reject) => {
+        try {
+            const url = window.location.href.replace(/\\/1$/, '');
+            const token = document.querySelector('meta[name="csrf-token"]').content.trim();
+            fetch(url, {
+                credentials: 'include',
+                headers: {
+                    'X-CSRF-TOKEN': token,
+                    'X-Requested-With': 'XMLHttpRequest',
+
+                },
+                 origin: window.location.origin,
+                 referrer: window.location.href,
+                 method: 'POST',
+            }).then(response => response.json()
+                .then(data => resolve(data.map(image => img_prt + '/' + image))));
+
+        } catch (error) {
+            reject(error);
+        }
+    });
+ `;
+
+@Common.MangaCSS(/^{origin}\/g\/[^/]+$/, 'h1.entry-title')
+@Common.MangasNotSupported()
+@Common.PagesSinglePageJS(pageScript)
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('akuma', 'Akuma.moe', 'https://akuma.moe', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Media.Manga, Tags.Language.Multilingual, Tags.Source.Aggregator, Tags.Rating.Pornographic);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        return [new Chapter(this, manga, new URL(`${manga.Identifier}/1`, this.URI).pathname, manga.Title)];
```

Comment:
Identifier seems to be semantically wrong, it points to the first page of the 'chapter.
Since there is no real chapter it should be consistent and use the manga identifier instead?

Reply:
The script to get pages has to be executed on this url. 

Had i used the common decorator (ChapterSinglePageCSS), the "chapter" link title would have been "Start reading" with no **clean** way to get the title in chapter extractor.

In any case i would need to override either FetchPages (to put the /1), either FetchChapters.